### PR TITLE
minimum integration tests

### PR DESCRIPTION
**Pull request by jeremymv2**
### Description

This adds minimum integration tests.

### Issues Resolved

n/a

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by jeremymv2** [comment]
I feel this is really necessary - since there are no integration tests running currently.

File: test/cookbooks/test_helper/.kitchen.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+driver:
```
Comment (chris-rock) [inline test/cookbooks/test_helper/.kitchen.yml]: I think we do not need this `.kitchen.yml` file

File: test/cookbooks/test_helper/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# test_helper
```
Comment (chris-rock) [inline test/cookbooks/test_helper/README.md]: lets remove this README

File: test/cookbooks/test_helper/chefignore [hunk 3]
```diff
@@ -0,0 +1,107 @@
+# Put files/directories that should be ignored in this file when uploading
```
Comment (chris-rock) [inline test/cookbooks/test_helper/chefignore]: Do we need that file?

**Comment by jeremymv2** [comment]
You are absolutely correct @chris-rock That was some un-needed clutter!  Cleaned up.

**Comment by chris-rock** [comment]
Awesome work @jeremymv2

**Pull request merged**